jib-core/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- `Ports.parse(List<String> ports)` in api now returns `Set` instead of `ImmutableSet` ([#2513](https://github.com/GoogleContainerTools/jib/pull/2513))
```

Comment:
```suggestion
- `Ports.parse(List<String> ports)` now returns `Set` instead of `ImmutableSet` ([#2513](https://github.com/GoogleContainerTools/jib/pull/2513))
```